developer/src/common/web/utils/src/xml-utils.ts

```diff
@@ -8,6 +8,8 @@
 
 import { XMLParser, XMLBuilder } from 'fast-xml-parser';
 
+export const START_INDEX = XMLParser.getStartIndexSymbol();
```

Comment:
As this turns into a global export on developer-utils, can we name it `XML_START_INDEX_SYMBOL`?

```suggestion
export const XML_START_INDEX_SYMBOL = XMLParser.getStartIndexSymbol();
```